manifests/profile/tools_lib/apache.pp

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::tools_lib::apache
+#
+# Configure Apache for tools.lib to reverse proxy atlassian suite
+# and serve a trivial splash page
+#
+# @example
+#   include nebula::profile::tools_lib::apache
+
+class nebula::profile::tools_lib::apache (
+  String $servername = 'tools.lib.umich.edu',
+) {
+
+  $docroot = '/srv/www'
+  $keyname = 'tools.lib.umich.edu'
+
+  file {
+    default:
+      ensure => 'directory',
+      mode   => '0755',
+    ;
+    $docroot:
+    ;
+    "${docroot}/rest":
+  }
+
+  $http_files = lookup('nebula::http_files')
+
+  file {
+    default:
+      ensure => 'present',
+      mode   => '0644',
+    ;
+    "${docroot}/index.html":
+      source => "https://${http_files}/tools.lib/index.html"
+    ;
+    "${docroot}/index.css":
+      source => "https://${http_files}/tools.lib/index.css"
+    ;
+    "${docroot}/mlibrary.png":
+      source => "https://${http_files}/tools.lib/mlibrary.png"
+    ;
+    "${docroot}/rest/index.html":
+      source => "https://${http_files}/tools.lib/rest/index.html"
+  }
+
+  $chain_crt = lookup('nebula::profile::ssl_keypair::chain_crt')
```

Comment:
Not needed -- that will be the default anyway

Reply:
This is actually used in the apache::vhost class further down the page. Moved, and removed the redundant injection into nebula::profile::ssl_keypair